src/test/java/th/ac/kmitl/science/comsci/example/CityNameTest.java

```diff
@@ -0,0 +1,16 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+import junit.framework.Assert;
+import org.junit.Test;
+public class CityNameTest {
+    String cname = "Bangkok";
```

Comment:
Use meaningful names for variables. The variable name must define the exact explanation of its content not this short `cname` but `cityName`.